lib/createTransport.js

```diff
@@ -13,10 +14,12 @@ function createTransport(options) {
 		keepAlive: true,
 	});
 
+	const limit = RateLimit(1);
```

Comment:
```suggestion
	const limit = RateLimit(18);
```
I _think_ the api rate limit is 20, but when importing lots of data that can sometimes lead to 'bunching', so best to set a little lower (see https://github.com/Financial-Times/biz-ops-github-importer/pull/26)
I think the limit set by the api is 10

Reply:
Because 18 is a magic number how about we go with 10 or 15? (basically the opposite of [adding 2 feet to Everest](https://en.wikipedia.org/wiki/Andrew_Scott_Waugh))

Reply:
Or using the `p-ratelimit` package we can specify the window as 5 seconds and limit as 100 as mentioned in the issue linked to?

Reply:
I agree 18 _feels _ like a magic number, and we should document it... but there is method in the madness.

Logs indicate that the number of times that an overspill to 21rps (due to variance in client to edge latency) when sending 20rps is about 10 times in every 1000 requests i.e 1% chance of a request getting rate limited

Therefore only need 1 weird latency event per 20 to get to 21rps and see errors if sending 20rps = 1% chance

But you need 2 per 19 if sending 19rps = 1% * 1% = 0.01% chance (1 in 10,000 requests)

You need _3_ per 18 if sending 18rps = 1% * 1% * 1% = 1 in 1,000,000 requests... which feels like good odds

> the window as 5 seconds and limit as 100

That would be worse I think as it would allow sending 100 requests in the first second. The comment in the original issue was to explain that when aggregating over larger intervals the general pattern seems to be ~20 rps, but something in the detail was busting it

![image](https://user-images.githubusercontent.com/447559/90229553-0270a080-de10-11ea-89fa-6a069970de4f.png)


vs

![image](https://user-images.githubusercontent.com/447559/90229584-10262600-de10-11ea-8488-04ec731dcbb8.png)

Same data, cut differently

Reply:
Thanks for the extra info, I've amended the limits and added comments with a link to this thread.